app/services/create_account.rb

```diff
@@ -41,6 +42,10 @@ def create_redis_namespace
     CreateRedisNamespaceJob.perform_later(account)
   end
 
+  def create_default_admin_set
+    CreateDefaultAdminSetJob.perform_later(account)
```

Comment:
The problem with this approach is that jobs are only allowed to fail a few times before being sent to the dead letter queue.  If your job hits that number before the solr or fedora jobs are done, your job will never complete it's work.